src/api/app/controllers/label_templates_controller.rb

```diff
@@ -0,0 +1,59 @@
+class LabelTemplatesController < ApplicationController
+  before_action :set_label_template, only: %i[update destroy]
+
+  after_action :verify_authorized # raise an exception if authorize has not yet been called.
+
+  # GET /label_templates
+  def index
+    authorize LabelTemplateGlobal
+    @label_templates = LabelTemplateGlobal.all
+
+    render 'label_templates/index', formats: [:xml]
+  end
+
+  # POST /label_templates
+  def create
+    authorize LabelTemplateGlobal
+    @label_template = LabelTemplateGlobal.new(label_template_params)
+
+    if @label_template.save
+      render_ok
+    else
+      render_error message: @label_template.errors.full_messages.to_sentence,
+                   status: 400, errorcode: 'invalid_label_template'
+    end
+  end
+
+  # PATCH/PUT /label_templates/1
+  def update
+    authorize @label_template
+
+    if @label_template.update(label_template_params)
+      render_ok
+    else
+      render_error message: @label_template.errors.full_messages.to_sentence,
+                   status: 400, errorcode: 'invalid_label_template'
+    end
+  end
+
+  # DELETE /label_templates/1
+  def destroy
+    authorize @label_template
+    @label_template.destroy
+
+    render_ok
+  end
+
+  private
+
+  def set_label_template
+    @label_template = LabelTemplateGlobal.find(params[:id])
```

Comment:
This might be nil.

Reply:
I'm trying to find when this could be nil, but I don't find a case.

For a non-existent global label template (say `id=42`), it will return:
```
<status code="not_found">
  <summary>Couldn't find LabelTemplateGlobal with 'id'=42</summary>
</status>
```
`find` returns `ActiveRecord::RecordNotFound` error if the object cannot be found. For reference: https://api.rubyonrails.org/v7.1.5.1/classes/ActiveRecord/Associations/CollectionProxy.html#method-i-find

Could you provide a case where this may be `nil`?

Reply:
> Could you provide a case where this may be nil?

Sorry, I was wrong (was thinking about `.find_by`). All good!